browser-test/src/support/admin_predicates.ts

```diff
@@ -108,7 +121,13 @@ export class AdminPredicates {
       },
     )
 
-    const valuesToSet = values ? values : value ? [value] : []
+    const valuesToSet = complexValues
+      ? complexValues
```

Comment:
I want this way to be easy to read but I agree it probably isn't :|  I tried adding !== undefined so it's more clear what the condition is, but that doesn't really help. I tried the null-coalesce operator (??) but that only works for the first two since there's no functional method to turn value into [{value: value}]. I tried the if/else way but I don't like declaring a variable and then setting it conditionally... it feels error-prone and repetitive, though it's probably fine in practice...

So I pulled it out to a method, which follows your suggestion without the declare/define separation and also adds some labeling that makes it clearer I think.